### PR TITLE
157 broken link checker work around

### PR DESCRIPTION
**Pull request by daveatweaviate**
[issue](https://github.com/weaviate/weaviate-edu-team-tasks/issues/157)
[staging](https://github.com/weaviate/weaviate-io/tree/157-broken-link-workaround)

### What's being changed:
work aroudn for broken links

### Type of change:
- [x] **Documentation** updates (non-breaking change to fix/update documentation)

### How Has This Been Tested?

- [x] **Github action** – automated build completed without build errors
- [x] **Local build** - the site works as expected when running `yarn start`

### CODE REVIEW

File: developers/weaviate/client-libraries/python.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ client = weaviate.Client(
 There is a variety of neural search frameworks that use Weaviate under the hood to store, search through, and retrieve vectors.
 
 - deepset's [haystack](https://www.deepset.ai/weaviate-vector-search-engine-integration)
-- Jina's [DocArray](https://docarray.jina.ai/advanced/document-store/weaviate/)
+- Jina's [DocArray](https://github.com/docarray/docarray)
```
Comment (daveatweaviate) [inline developers/weaviate/client-libraries/python.md:163]: These 3d party docs seem to have moved today.

**Review by daveatweaviate — COMMENTED** [review]
Reviewer notes

**Comment by daveatweaviate** [comment]
@databyjp this is ready for review when you have a moment

File: README.md [hunk 2]
```diff
@@ -147,6 +147,16 @@ The `build` command is useful when you are finished editing. If you ran
...
 see you changes while you are editing.
 
+The build command runs a link checker. If you are having trouble with temporarily broken links, you can update the `URL_IGNORES` variable to disable checking for that link.
+
+To disable link checking, add the broken URL to the `URL_IGNORES` lists in these scripts:
+
+- [verify-links.sh](https://github.com/weaviate/weaviate-io/blob/main/_build_scripts/verify-links.sh)
+- [verify-links-build-dev.sh](https://github.com/weaviate/weaviate-io/blob/main/_build_scripts/verify-links-build-dev.sh)
+
+Be sure to restore normal link checking before creating a pull request.
```
Comment (databyjp) [inline README.md:157]: Maybe
```Please remove any unnecessary links as from `URL_IGNORES` they become restored.```
?

**Review by daveatweaviate — COMMENTED** [review]
@databyjp thanks for the feedback, updated as per slack

This is ready for another look

**Pull request merged**